core/src/main/resources/jenkins/fingerprints/GlobalFingerprintConfiguration/config.jelly

```diff
@@ -19,11 +19,14 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
-    <j:if test="${descriptor.fingerprintStorageDescriptors.size() gt 1}">
-        <f:section title="${%Fingerprints}">
+    <f:section title="${%Fingerprints}">
+        <f:entry title="Disable Fingerprint Cleanup">
+            <f:checkbox field="fingerprintCleanupDisabled" default="${it.fingerprintCleanupDisabled}" />
```

Comment:
Is there any reason to disable this if there isn’t an external plugin for fingerprints?

Reply:
It might be a good feature to have in case the disk storage is cheaper for a user than having a periodically running process. Or say Jenkins is configured on a distributed disk storage. I don't have any strong opinions though :)